src/custom-element.js

```diff
@@ -830,6 +848,9 @@ function createBaseCustomElementClass(win) {
         // Resources can now be initialized since the ampdoc is now available.
         this.resources_ = Services.resourcesForDoc(this.ampdoc_);
       }
+      if (!this.owners_) {
```

Comment:
i'd not have it as a member field. i hope we can remove its only usage in L1632, but fine for now.